docs/website/content/2.configuration/4.block/settings.md

```diff
@@ -12,6 +12,12 @@ values: [
     "name": "accept_forwarded_url",
     "type": "tuple (string)"
   },
+  {
```

Comment:
Add new settings to docs/website/content/2.configuration/0.command-line.md, too.

Reply:
Done